### PR TITLE
[tests-only] Add api test to check files and folders datetime format

### PR DESCRIPTION
**Pull request by saw-jan**
## Description

oC implements RFC's IMF-fixdate format https://tools.ietf.org/html/rfc7231#section-7.1.1.1
## Related Issue
- https://github.com/owncloud/ocis/issues/1154

## How Has This Been Tested?
- test environment: local

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
Lgtm :+1:

**Review by Talank — COMMENTED** [review]
lgtm

File: tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature [hunk 1]
```diff
@@ -272,6 +272,7 @@ Feature: files and folders exist in the trashbin after being deleted
       | old      |
       | new      |
 
+  @issue-ocis-541
```
Comment (individual-it) [inline tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature:275]: what does this PR has to do with issue 541? I though it's about 1154

File: tests/acceptance/features/apiWebdavUpload1/uploadFile.feature [hunk 2]
```diff
@@ -199,3 +199,15 @@ Feature: upload file
...
       | old         |
       | new         |
+
+  @issue-ocis-1154
+  Scenario Outline: files and folders have standard datetime
+    Given using <dav_version> DAV path
+    And user "Alice" has created folder "testFolder"
+    And user "Alice" has uploaded file with content "first time upload content" to "file.txt"
+    When user "Alice" gets all information about files and folders from "/" directory using the WebDAV API
+    Then the files and folders should have the standard datetime format in the last response
```
Comment (individual-it) [inline tests/acceptance/features/apiWebdavUpload1/uploadFile.feature:209]: no need to get all information, we are only interested in the time here.
Maybe we even don't need a new test, what about https://github.com/owncloud/core/blob/c1eb06384b1303cf72ae29fb5d8aed0125f1eaf6/tests/acceptance/features/apiWebdavProperties2/getFileProperties.feature#L291-L302 isn't that testing exactly what we need?

**Review by individual-it — CHANGES_REQUESTED** [review]

**Comment by saw-jan** [comment]
This PR does the same test as mentioned by @individual-it so it is not necessary to add same kind of test twice.
> no need to get all information, we are only interested in the time here.
> Maybe we even don't need a new test, what about
> 
> https://github.com/owncloud/core/blob/c1eb06384b1303cf72ae29fb5d8aed0125f1eaf6/tests/acceptance/features/apiWebdavProperties2/getFileProperties.feature#L291-L302
> 
> isn't that testing exactly what we need?

**Pull request closed without merging**